cmd/oras/manifest/push.go

```diff
@@ -135,6 +149,15 @@ func pushManifest(opts pushOptions) error {
 		}
 	}
 
+	display.Print("Pushed", opts.targetRef)
+
+	var tagBytesNOpts oras.TagBytesNOptions
```

Comment:
Should be based on the default options.
```suggestion
	tagBytesNOpts := oras.DefaultTagBytesNOptions
```

Reply:
done